src/index.html

```diff
@@ -7,8 +7,43 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,500;1,700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html">
+        <img
+        src="./images/logo.png"
+        alt="logo"
+        class="logoImg"
+      >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__items"><a href="index.html" class="nav__links is-active">Apple</a></li>
```

Comment:
1. Why is it `nav__items`? I think it is better to rename it like `nav__item` (same with `nav__links`)
2. Add empty lines between multiline sibling blocks of HTML ([checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md))